sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_encode.py

```diff
@@ -569,7 +569,7 @@ def encode_application_properties(value):
         return {TYPE: AMQPTypes.null, VALUE: None}
     fields = {TYPE: AMQPTypes.map, VALUE: cast(List, [])}
     for key, data in value.items():
-        cast(List, fields[VALUE]).append(({TYPE: AMQPTypes.string, VALUE: key}, data))
+        cast(List, fields[VALUE]).append(({TYPE: AMQPTypes.string, VALUE: key}, data.decode("utf-8") if type(data) == bytes else data))
```

Comment:
I don't think we should make this change this low down - the payload encoding can't assume that any bytes passed in were intended to be strings - I would move this up to the ServiceBus layer.

Reply:
Ill look at moving it up there, my only concern was how that would affect the ServiceBus layer when its working with uamqp and would need the strings to be in bytes to cross over.